rclcpp_examples/test/listener.py

```diff
@@ -0,0 +1,57 @@
+import io
+from launch.output_handler import LineOutput
+
+
+class CheckerHandler(LineOutput):
+    EXPECTED_LAST_LINE = b'I heard: [Hello World: 2]'
```

Comment:
Or just check that the listener receives `b'I heard: [Hello World: %d]'`, and not worry about the number?